imodels/rule_set/slipper.py

```diff
@@ -0,0 +1,282 @@
+import numpy as np
+import random
+
+from joblib import Parallel, delayed
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.model_selection import train_test_split
+
+from imodels.util.rule import Rule
+
+
+class SlipperRule(BaseEstimator, ClassifierMixin):
```

Comment:
Could you rename this to SlipperClassifier to be consistent with everything else (even if it's mainly just used by boosted_rules)?